src/cli/upload.rs

```diff
@@ -39,7 +41,7 @@ pub async fn execute(args: Args) -> miette::Result<()> {
         HumanBytes(filesize)
     );
 
-    let client = default_authenticated_client();
+    let client = AuthenticatedClient::from_client(Client::new(), Default::default());
```

Comment:
I think you can just do `AuthenticatedClient::default` !

Reply:
Indeed!